README.md

```diff
@@ -10,24 +10,28 @@
     Maintained by <a href="https://www.zakodium.com">Zakodium</a>
   </p>
 
-
 [![NPM version][npm-image]][npm-url]
 [![Test coverage][codecov-image]][codecov-url]
 [![npm download][download-image]][download-url]
 
 </h3>
 
-
-A library for encoding bmp image file format.
+A library for encoding bmp image file format with `BITMAPV5HEADER`. Specifications taken from
```

Comment:
Its for encoding AND decoding.
I suggest you say in the supported feature below that it only supports the V5 header format.